src/sentry/static/sentry/app/components/activity/note/input.jsx

```diff
@@ -60,9 +59,9 @@ class NoteInput extends React.Component {
   constructor(props) {
     super(props);
 
-    const {item} = props;
-    const existing = !!item;
-    const defaultText = existing ? item.data.text || '' : props.defaultText;
+    const {item, text} = props;
+    const existing = !!item && !!item.id;
```

Comment:
Should the `id` be passed in as `itemID` to match the `text` prop? That would allow the `NoteInput` to have even less context on what was having a note created.